src/Components/Confirmation.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Confirmation = (props) => {
+    return (
+        <div>
+            <h2>Thank you for your order!</h2>
+            {Object.keys(props.order).map(property => {
```

Comment:
I like your use of Object.keys. This saved a lot of time writing the component display your pizza items. I'll need to keep this in mind more in the future.